cluster/index.go

```diff
@@ -135,6 +137,7 @@ func (index *Index) EachPod(pod PodInfo) {
 					index.LinkedConfigMaps.Add(fmt.Sprintf("%s/%s", cm.Namespace, cm.Name))
 				}
 			}
+			index.PodStatus.Add(pod.Status)
```

Comment:
Sorry this should actually be lifted up to the top level and not in this owner loop as it only counts the DaemonSets here. :/